src/week01/day1/index.rst

```diff
@@ -45,6 +45,8 @@ Exercises
 Studio
 ------
 
+* :ref:`junit-studio`
+
 Studio Setup
```

Comment:
Should the **Studio Setup**  and the **Turning in the Studio** be moved into the Studio itself?